mesonbuild/modules/gnome.py

```diff
@@ -886,6 +885,107 @@ def mkenums(self, state, args, kwargs):
         else:
             return ModuleReturnValue(targets, targets)
 
+    def mkenums_simple(self, state, args, kwargs):
+        hdr_filename = args[0] + '.h'
+        body_filename = args[0] + '.c'
+
+        # not really needed, just for sanity checking
+        forbidden_kwargs = ['c_template', 'h_template', 'eprod', 'fhead',
```

Comment:
The main purpose of this was to catch old-api arguments from people who migrate from `mkenums` to `mkenums_simple`, to make sure there is no leakage. I can also remove this entirely. Our implementation calls the `mkenums` implementation in the end, so things will be checked there too.